xmtp/src/conversations.rs

```diff
@@ -44,12 +44,42 @@ where
         Self { client }
     }
 
-    pub async fn new_secret_conversation(
+    pub fn new_secret_conversation(
```

Comment:
Is there a distinction between secret_conversations and conversations?

Reply:
It's the only conversation type we have right now. Legacy of padlock